packages/now-next/src/index.ts

```diff
@@ -280,35 +280,35 @@ export const build = async ({
   }
 
   if (process.env.NPM_AUTH_TOKEN) {
-    debug('found NPM_AUTH_TOKEN in environment, creating .npmrc');
+    debug('Found NPM_AUTH_TOKEN in environment, creating .npmrc');
     await writeNpmRc(entryPath, process.env.NPM_AUTH_TOKEN);
   }
 
-  debug('installing dependencies...');
+  console.log('Installing dependencies...');
```

Comment:
@sophearak Was this changes to console.log() intentionally?

Reply:
It looks like you changed it in every builder so I think that's good 👍

Reply:
Indeed, this is intended 🙌